flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/MySqlSourceTest.java

```diff
@@ -584,6 +585,8 @@ public void go() throws Exception {
     }
 
     @Test
+    @Ignore(
+            "The test failed when gtid mode enabled, we should use offset map for specific-offset startup mode")
```

Comment:
We already support offset map in this PR, is this case still not work?

Reply:
This case is not releated the code change , but caused by the mysql docker enabled gtid mode.
It failed in legacy source, the failed reason is we snapshot the offset in a map<String, String> which contains
`gtid set`, but the test only use the `binlog fileName` and `binlog position` in this test, this will lead the debezium engine comapring offset failing.

Reply:
Please create an issue to track the `specific-offset` problem when it works in gtid mode. And still use non-gtid mode for this test.

Reply:
#433